formal-spec/participants-vs-elapsed_time.dat

```diff
@@ -1,6 +1,6 @@
-1.00E+02	4.01E-06
-1.00E+03	4.23E-05
+1.00E+02	4.04E-06
+1.00E+03	4.22E-05
 1.00E+04	4.53E-04
-1.00E+05	6.70E-03
-1.00E+06	7.50E-02
-1.00E+07	2.92E+00
+1.00E+05	7.55E-03
+1.00E+06	8.37E-02
```

Comment:
@nkarag does this mean that the processing time went up by ~100 milliseconds, for 1M participants, but went down by 1 second for 10M participants?

Reply:
Yeah. That is what it seems. These are the numbers!